common/source/docs/common-polygon_fence.rst

```diff
@@ -1,13 +1,14 @@
 .. _polygon_fence:
 
+[copywiki destination="copter,rover"]
 =============
 Polygon Fence
 =============
 
 Overview
 ========
 
-Copter 3.4 (and higher) include support for a polygon fence with up to 84 points.  The purpose of this fence is to attempt to stop your vehicle from flying out of the polygon by initiating an RTL or, if flying in Loiter mode, the vehicle will normally stop before breaching the fence.  This feature is an extension of the simpler :ref:`circular fence <ac2_simple_geofence>`.
+Copter 3.4 (and higher) include support for a polygon fence with up to 84 points.  The purpose of this fence is to attempt to stop your vehicle from flying out of the polygon by initiating an RTL or, if flying in Loiter mode, the vehicle will normally stop before breaching the fence.  This feature is an extension of the simpler :ref:`circular fence <common-ac2_simple_geofence>`.
```

Comment:
as discussed it's 70 polygon fence points apparently

Reply:
done